profiles/base/crd.yaml

```diff
@@ -10,7 +10,7 @@ spec:
   names:
     kind: Profile
     plural: profiles
-  scope: Namespaced
+  scope: Cluster
```

Comment:
about this change, we had discussed in https://github.com/kubeflow/manifests/pull/29#discussion_r277858777
@kkasravi WDYT?

Reply:
Latest CRD on controller side is Cluster scope:
https://github.com/kubeflow/kubeflow/blob/master/components/profile-controller/config/crds/kubeflow_v1alpha1_profile.yaml